scripts/ci/travis-check.js

```diff
@@ -91,18 +141,22 @@ async function testBuild( lang ) {
  * @param {String} cwd
  * @param {String} command Program to execute.
  * @param {Array.<String>} modifiers Program's modifiers.
+ * @param {String} stderr Optional alternative output stream for stderr.
```

Comment:
* If this value is optional, it should be marked with square brackets.
	```
	@param {String} [stderr] ...
	```
* If it can be a stream, why is it specified as a string?

According to https://nodejs.org/api/child_process.html#optionsstdio, the definition should look a bit different.

```suggestion
 * @param {'pipe' | 'overlapped' | 'ipc' | 'ignore' | 'inherit' | Stream | Number | null} [stderr] Optional alternative output stream for stderr.
```

Or a simplified version:

```js
/**
 * @param {Object} options
 * @param {Boolean} [options.pipeStderr=false]
 */
function executeCommand( cwd, command, modifiers, options = {} ) {
```